### ISSUE TITLE
hasTier cannot be a subproperty of substring and superstring at the same time

### ISSUE THREAD

**Issue by Glottotopia** [acoli-repo/ligt#2]
Utterances can point to tiers in a meronymic relationship, but the meaning is "U has subpart of type tier T". 
Words can point to tiers in a meronymic relationship, but the meaning is "W is itself part of a structure of type tier T". 
The two properties are different and should not use the same label.

**Comment by chiarcos** [comment]
Note that tier is modelled as a string. This may be counter-intiuitive to a linguist, but this is necessary because of the anchoring of the vocabulary in NIF.

As a relation between strings, hasTier can be subproperty of both *if tier and utterance have exactly the same extension* (i.e., exactly one tier instance per utterance). This is the intended interpretation. Equivalent tier instances of the same tier category (e.g., MorphTier) can be made explicit by hasTier subproperties and Tier subclasses.

As for the ontological status of tier, indeed, I assumed a different interpretation of "tier" when creating the first Ligt draft: Because of the anchoring in NIF, tiers, utterances and words are all strings, so this is just a re-labelled hasSuperString property, and the tier is a subtype of string.

A better way of modelling would be to model tiers independently from NIF strings. I suggest to discuss this in the context of https://github.com/ld4lt/linguistic-annotation, and to revise Ligt afterwards.

As for the definition of tier: I'm open for another way of modelling already now, but only on the basis of a scientific publication or a technical whitepaper. If you have anything at hand, we should open the issue again.

**Comment by chiarcos** [comment]
To be discussed before advancement to next version.

**Comment by Glottotopia** [comment]
Just a side note: there can be cases where the words used for glossing do not neatly map on the string in the source line. 
```
haste      nich  gesehen
hast=du    nicht ge-seh-en
have=2ps   NEG   PTCP-see-PTCP
```
In this case, "du" is not a substring of the source line string. Nevertheless, there is a semantic relation between lines 1 and 2 which one would want to capture. At least for those cases, a mechanism different from NIF would have to be used AFAICS. The question is whether this other mechanism should then also be used for vanilla cases as well.

**Comment by max-ionov** [comment]
Yes, this is an issue, and, in fact, it is happening very often due to morphophonology. In the current version of the vocabulary, we decided to use it anyway to be able to keep the connection with NIF. Conceptually, it _is_ a substring, even when it is not on the surface.

**Comment by chiarcos** [comment]
> Just a side note: there can be cases where the words used for glossing do not neatly map on the string in the source line.
> 
> ```
> haste      nich  gesehen
> hast=du    nicht ge-seh-en
> have=2ps   NEG   PTCP-see-PTCP
> ```
> 
> In this case, "du" is not a substring of the source line string. Nevertheless, there is a semantic relation between lines 1 and 2 which one would want to capture. At least for those cases, a mechanism different from NIF would have to be used AFAICS. The question is whether this other mechanism should then also be used for vanilla cases as well.

see modelling in samples/nordhoff-1.ttl and comments for upcoming ligt v.0.3 version under experimental/
